src/ensembl/src/content/app/contact-us/ContactUs.scss

```diff
@@ -0,0 +1,43 @@
+.wrapper {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 1px;
+    z-index: 11; /* Need to go above the top bar */
+  }
+
+  .panelWrapper{
+      position: absolute;
+      right: 0;
+      top: 90px;
+      bottom: 140px;
```

Comment:
How do we deal with small screens?

![image](https://user-images.githubusercontent.com/6834224/127142588-1768a961-1f1f-4a28-be72-9e6250491e0f.png)